### ISSUE TITLE
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.

### ISSUE THREAD

**Issue by legluondunet** [lutris/lutris#3251]
Hello Lutris dev team,
I use Ubuntu 20.04 with gamemode installed:

```
$ dpkg -l gamemode
Souhait=inconnU/Installé/suppRimé/Purgé/H=à garder
| État=Non/Installé/fichier-Config/dépaqUeté/échec-conFig/H=semi-installé/W=attend-traitement-déclenchements
|/ Err?=(aucune)/besoin Réinstallation (État,Err: majuscule=mauvais)
||/ Nom            Version          Architecture Description
+++-==============-================-============-===========================================
ii  gamemode       1.5.1-0ubuntu3.1 amd64        Optimise Linux system performance on demand
```
```

$ dpkg -L gamemode
/.
/etc
/etc/gamemode.ini
/usr
/usr/bin
/usr/bin/gamemoded
/usr/bin/gamemoderun
/usr/lib
/usr/lib/systemd
/usr/lib/systemd/user
/usr/lib/systemd/user/gamemoded.service
/usr/libexec
/usr/libexec/gamemode-simulate-game
/usr/share
/usr/share/dbus-1
/usr/share/dbus-1/services
/usr/share/dbus-1/services/com.feralinteractive.GameMode.service
/usr/share/doc
/usr/share/doc/gamemode
/usr/share/doc/gamemode/README.Debian
/usr/share/doc/gamemode/README.md.gz
/usr/share/doc/gamemode/TODO.Debian
/usr/share/doc/gamemode/copyright
/usr/share/lintian
/usr/share/lintian/overrides
/usr/share/lintian/overrides/gamemode
/usr/share/man
/usr/share/man/man8
/usr/share/man/man8/gamemoded.8.gz
/usr/share/polkit-1
/usr/share/polkit-1/actions
/usr/share/polkit-1/actions/com.feralinteractive.GameMode.policy
/usr/share/doc/gamemode/changelog.Debian.gz
```
Each time I launch a game with Lutris I obtain gamemode error messages:

```
DEBUG    2020-11-03 01:20:41,085 [command.start:132]:/home/romain/Autres_applications/Jeux/Lutris/lutris/share/lutris/bin/lutris-wrapper Tomb Raider II - GOG 0 0 gamemoderun /home/romain/.local/share/lutris/runners/wine/wine-staging-5.7-x86_64/bin/wine /home/romain/Games/tomb-raider-ii/drive_c/windows/syswow64/wineconsole.exe c:\TR2M.BAT
Running gamemoderun /home/romain/.local/share/lutris/runners/wine/wine-staging-5.7-x86_64/bin/wine /home/romain/Games/tomb-raider-ii/drive_c/windows/syswow64/wineconsole.exe c:\TR2M.BAT
Initial process has started with pid 6126
Start monitoring process.
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object '/usr/$LIB/libgamemodeauto.so.0' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
user.reg:3348: Malformed value '"HDA NVidia HDMI/DP,pc"=3 (js)"="disabled"'
```

**Comment by stephanlachnit** [comment]
Can you run `gamemoded -t` and `apt list *gamemode*`?

**Comment by legluondunet** [comment]
as requested:

```
$ gamemoded -t
: Loading config
Loading config file [/etc/gamemode.ini]
: Running tests

:: Basic client tests
:: Passed

:: Dual client tests
gamemode request succeeded and is active
Quitting by request...
:: Passed

:: Gamemoderun and reaper thread tests
...Waiting for child to quit...
...Waiting for reaper thread (reaper_frequency set to 5 seconds)...
:: Passed

:: Supervisor tests
:: Passed

:: Feature tests
::: Verifying CPU governor setting
::: Passed
::: Verifying Scripts
::: Passed (no scripts configured to run)
::: Verifying GPU Optimisations
::: Passed (gpu optimisations not configured to run)
::: Verifying renice
::: Passed (no renice configured)
::: Verifying ioprio
::: Passed
:: Passed

: All Tests Passed!
```
```
$ apt list *gamemode*
En train de lister... Fait
gamemode/focal-updates,now 1.5.1-0ubuntu3.1 amd64  [installé]
gnome-shell-extension-gamemode/focal,focal 4-2 all
libgamemode-dev/focal-updates 1.5.1-0ubuntu3.1 amd64
libgamemode0/focal-updates,now 1.5.1-0ubuntu3.1 amd64  [installé, automatique]
libgamemodeauto-dev/focal-updates 1.5.1-0ubuntu3.1 amd64
libgamemodeauto0/focal-updates,now 1.5.1-0ubuntu3.1 amd64  [installé, automatique]
```

**Comment by stephanlachnit** [comment]
Looks like you're missing the 32bit libs.

**Comment by legluondunet** [comment]
Ubuntu 20.04 does not provide gamemode 32 bits packages, do you think this error message is really related to gamemode 32 bits packages?

**Comment by legluondunet** [comment]
@stephanlachnit, as you adviced in this issue:
https://github.com/FeralInteractive/gamemode/issues/254#issuecomment-643648779
I uninstalled Ubuntu gamemode packages and installed Debian ones which provides gamemode 32 packages too.
Now Lutris gamemode error messages disappeared:
```

...
Start monitoring process.
gamemodeauto: 
user.reg:1024: Malformed value '"HDA NVidia HDMI/DP,pc"=3 (js)"="disabled"'
...
```

**Comment by stephanlachnit** [comment]
Alright nice, then this is fixed. The `user.reg` warning does not come from gamemode afaict.

**Comment by strycore** [comment]
this is not our bug.
just run wine by itself with gamemode to experience.

**Comment by peepopoggers** [comment]
Hi, I had this issue and I installed the debian repos (for reference I am using linux mint 20.3) and the error disappeared, however, after that my .exe launched and closed immediately without any errors. I have tried with multiple wine versions in lutris, and all I get is:

```
Monitored process exited.
Initial process has exited (return code: 0)
Exit with return code 0
```

Is there any way to get more information to why the monitored process exited? Thanks.
